R/step-nest.R

```diff
@@ -0,0 +1,90 @@
+#' Nest
+#'
+#' @description
+#' This is a method for the tidyr `nest()` generic.
```

Comment:
```suggestion
#' This is a method for the [tidyr::nest()] generic.
```

Add brief description of translation?